Assets/Editor/MapzenMapEditor.cs

```diff
@@ -83,6 +86,63 @@ private void TileDataFoldout()
         EditorGUI.indentLevel--;
     }
 
+    private void UnitScaleToggle(MapzenMap mapzenMap, RegionScaleUnits.Units unit)
+    {
+        bool isSet = RegionScaleUnits.Test(unit, mapzenMap.RegionScaleUnit);
+        isSet = EditorGUILayout.Toggle(unit.ToString(), isSet);
+
+        mapzenMap.RegionScaleUnit = isSet ? unit : mapzenMap.RegionScaleUnit;
+    }
+
+    private void RegionScaleRatioFoldout()
+    {
+        float unitConverter = 1.0f;
+        showRegionScaleRatioFoldout = EditorGUILayout.Foldout(showRegionScaleRatioFoldout, "Region Scale Ratio");
+        if (!showRegionScaleRatioFoldout)
+        {
+            return;
+        }
+
+        mapzenMap.RegionScaleRatio = EditorGUILayout.FloatField("Scale: ", mapzenMap.RegionScaleRatio);
+        GUILayout.Label("Choose world scale units: ");
```

Comment:
In the future we might want to give a little more descriptive panel with a mouse over for this, it could be misleading to have those two values with a similar naming.